src/tasks/leveling.ts

```diff
@@ -1192,7 +1321,7 @@ export const LevelingQuest: Quest = {
     {
       name: "Powerlevel",
       completed: () =>
-        myBasestat($stat`Mysticality`) >= targetBaseMyst - targetBaseMystGap &&
+        myBasestat(myPrimestat()) >= targetBaseMyst - targetBaseMystGap && // I don't know if this will cause issues.
```

Comment:
This is fine, we can test it and update it as necessary, but leave the comments out of the code